src/ISFileManager.cpp

```diff
@@ -462,7 +462,7 @@ namespace ISFileManager {
     std::string CurrentWorkingDirectory()
     {
         char curdir[256] = { 0 };
-        if (_GETCWD(curdir, sizeof(curdir)) != nullptr)
+        if (_GETCWD(curdir, sizeof(curdir)) != 0)
```

Comment:
Was there a reason to change this from nullptr?  Technically, nullptr is correct, since this may result in warnings about type casting.